assets/src/edit-story/components/panels/design/noSelection/noSelection.js

```diff
@@ -24,22 +24,21 @@ import { __ } from '@web-stories-wp/i18n';
  * Internal dependencies
  */
 import { Panel, PanelContent } from '../../panel';
+import { Text, THEME_CONSTANTS } from '../../../../../design-system';
 
-const Note = styled.p`
-  font-style: italic;
-  font-family: ${({ theme }) => theme.DEPRECATED_THEME.fonts.label.family};
-  font-weight: ${({ theme }) => theme.DEPRECATED_THEME.fonts.label.weight};
-  font-size: ${({ theme }) => theme.DEPRECATED_THEME.fonts.label.size};
-  line-height: ${({ theme }) => theme.DEPRECATED_THEME.fonts.label.lineHeight};
-  margin: 50px 0;
+const Note = styled(Text)`
+  color: ${({ theme }) => theme.colors.fg.secondary};
   text-align: center;
+  margin: 150px 0;
```

Comment:
We can't use the full space? Could you ask Sam how she wants this to move in the space with various window heights and with the layer panel open and closed?

Reply:
I was wondering about it myself, too. It's not really positioned in some specific way there (e.g. it's not in the middle). Since the panel takes the height its content has then this was just the easiest way to still look fine with different window heights. Not sure what would be the reference max-height for this.

Will ask Sam then.